public/app/plugins/datasource/grafana-testdata-datasource/.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules/
+.yarn/.cache/
```

Comment:
Aren't these needed? [yarn](https://github.com/grafana/grafana/blob/main/.gitignore#L23) and [node_modules](https://github.com/grafana/grafana/blob/main/.gitignore#L1) are already referenced in the root gitignore file? Maybe we just add the `dist/` to root gitignore and remove this file entirely?

Would be nice to update the plugin-config webpack config so the cache files that are created in these directories are moved to the same directory as the grafana core cache files.

Reply:
> Would be nice to update the plugin-config webpack config so the cache files that are created in these directories are moved to the same directory as the grafana core cache files.

I will check that as a follow-up PR as an improvement.